src/main.cc

```diff
@@ -2112,24 +2108,7 @@ SquidShutdown()
     storeLogClose();
     accessLogClose();
     Store::Root().sync();       /* Flush log close */
-    StoreFileSystem::FreeAllFs();
     DiskIOModule::FreeAllModules();
-#if LEAK_CHECK_MODE && 0 /* doesn't work at the moment */
-
-    configFreeMemory();
-    storeFreeMemory();
-    /*stmemFreeMemory(); */
-    netdbFreeMemory();
-    ipcacheFreeMemory();
-    fqdncacheFreeMemory();
-    asnFreeMemory();
-    clientdbFreeMemory();
-    statFreeMemory();
-    eventFreeMemory();
-    mimeFreeMemory();
```

Comment:
> I would like the LEAK_CHECK_MODE removal as s separate PR. It is one of those feature-like changes that need highlighting for those who may be trying to use it.

Will extract. Please note that the LEAK_CHECK_MODE feature has been disabled since 2006 commit afec404. It cannot be enabled without changing Squid source code, including dealing with the no longer globally available mimeFreeMemory() that was hidden since 2012 commit b65ce00.

Reply:
> besides resolving trivial code conflicts, this PR description will need to be updated to refer to the rationale partially documented in PR 1130

Done (after PR 1130 got merged).